src/main/java/io/nats/client/api/PublishAck.java

```diff
@@ -36,6 +37,7 @@ public PublishAck(Message msg) throws IOException, JetStreamApiException {
         if (stream == null) {
             throw new IOException("Invalid JetStream ack.");
         }
+        domain = JsonUtils.readString(json, DOMAIN_RE, null);
```

Comment:
I assume that your utils will not throw an exception if the field is not found, right?

Reply:
yeah, readString handles that